R/bundleFiles.R

```diff
@@ -179,7 +179,7 @@ ignoreBundleFiles <- function(dir, contents) {
     # rsconnect packages
     "rsconnect", "rsconnect-python", "manifest.json",
     # packrat + renv,
-    "renv", "renv.lock", "packrat",
+    "renv", "packrat",
```

Comment:
Yes, but I don't think that should matter because the settings only really affect interactive usage, and the consequence of those settings will be encoded in the lockfile.

Reply:
The recommendation to use `renv::settings$snapshot.type("explicit")` presents a challenge, then, if we expect folks to set that ahead of each deploy operation but do so in a way that does not modify their existing renv project settings. What is your suggested workflow?